src/Symfony/Component/Form/Resources/translations/validators.es.xlf

```diff
@@ -14,6 +14,126 @@
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Este valor no es un color HTML5 válido.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Por favor, ingrese una fecha de cumpleaños válida.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>La opción seleccionada no es válida.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>La colección no es válida.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Por favor, seleccione un color válido.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Por favor, seleccione un país válido.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Por favor, seleccione una moneda válida.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Por favor, elija un intervalo de fechas válido.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Por favor, ingrese una fecha y hora válidas.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Por favor, ingrese una fecha valida.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Por favor, seleccione un archivo válido.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>El campo oculto no es válido.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Por favor, ingrese un número entero.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Por favor, seleccione un idioma válido.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Por favor, seleccione un lugar válido.</target>
```

Comment:
@Nyholm while checking this: https://github.com/symfony/symfony/pull/38721#discussion_r511602598,
I realized that this translation for 'locale' is for 'location' as well.
In Spanish, and in this context, 'locale' refers to 'configuración regional de idioma' = 'language locale'.
is it ok if i interpret this as 'Please select a valid language'?

Reply:
No, that would be wrong. A locale is rather some region-specific setting. It includes language but also formatting of numbers, dates, time, currency etc. as well as string collations and character case conversions.

Reply:
So a locale is "sv", "en", "es" etc. 
A language is "Swedish", "English", "Spanish". 

The word "locale" is a technical word, that is fine.

In the Arabic language, there is apparently no word for "locale", so we fallback to "language". I would much prefer if we used the Spanish word for "locale" here. 

Using "location", (Stockholm, "at work" or "London") is the wrong word in this context.

Reply:
@derrabus OK! thanks! if that's the case the correct translation would be 'Por favor, seleccione una configuración regional válida'.

Reply:
Yes. I agree. Can you please update?

Reply:
OK. Just to be clear about the changes:
If we talk about 'configuración regional' i would expect to see something like 'es_ES' = 'Spanish Spain' or 'es_CO' = 'Spanish Colombia', and if we talk about 'language' it is only 'es' = 'Spanish'.
i will update it.

Reply:
OK. Just to be clear about the changes:
If we talk about 'configuración regional' i would expect to see something like 'es_ES' = 'Spanish Spain' or 'es_CO' = 'Spanish Colombia', and if we talk about 'idioma' it is only 'es' = 'Spanish'.
i will update it.

Reply:
That is way too much Spanish for me (even with Google translate) I'll let someone else give their opinion. 

But both are better alternatives that the current one.

Reply:
ok, i'll wait for more feedback

Reply:
@TavoNiievez please, use this translation recommended by Wikipedia -> https://es.wikipedia.org/wiki/Configuraci%C3%B3n_regional

Reply:
@javiereguiluz thanks you! Done. 😃